test/game/tictactoe_test.exs

```diff
@@ -2,59 +2,24 @@ defmodule TicTacToeTest do
   use ExUnit.Case
   doctest TicTacToe
 
-  @empty_board Board.create(3)
-  @players ["x", "o"]
+  @board Board.create(2)
 
-  test "asks for spaces until board is full" do
-    fake_IO = spawn(FakeDevice, :receive_input, [["1", "2", "3"]])
+  test "accepts chooser definition" do
+    player_one = Player.new("X", &fake_chooser_one/2)
+    player_two = Player.new("O", &fake_chooser_two/2)
 
-    updated_board = TicTacToe.start(@players, @empty_board, fake_IO, FakeScreen)
+    players = [player_one, player_two]
 
-    assert Board.get(updated_board, 1) == {:ok, "x"}
-    assert Board.get(updated_board, 2) == {:ok, "o"}
-    assert Board.get(updated_board, 3) == {:ok, "x"}
-  end
-
-  test "asks again if chosen space is occupied" do
-    fake_IO = spawn(FakeDevice, :receive_input, [["1", "1", "2", "3"]])
-
-    updated_board = TicTacToe.start(@players, @empty_board, fake_IO, FakeScreen)
-
-    assert Board.get(updated_board, 1) == {:ok, "x"}
-    assert Board.get(updated_board, 2) == {:ok, "o"}
-    assert Board.get(updated_board, 3) == {:ok, "x"}
-  end
+    updated_board = TicTacToe.play_game(@board, players)
 
-  test "asks again if chosen space does not exist" do
```

Comment:
eliminaste un par de pruebas y funcionalidad
- asks again if chosen space does not exist
- asks again if chosen space is occupied